mitodl/micromasters
# Learners in Program Card

#### What are the relevant tickets?
Fix #2516

#### What's this PR do?
Add Learners in The Program card to the dashboard.

#### How should this be manually tested?
Make sure there are at least 9 or 10 learners in the program you are enrolled, that have profile images and their profiles are not private. Reload the page several times to see if the selected 8 profiles change randomly.
